modules/avatar/react/spec/Avatar.spec.tsx

```diff
@@ -29,7 +29,13 @@ describe('Avatar Accessibility', () => {
 
   test('Avatar should pass axe DOM accessibility guidelines', async () => {
     const html = ReactDOMServer.renderToString(<Avatar />);
-    expect(await axe(html)).toHaveNoViolations();
+    expect(
+      await axe(html, {
+        rules: {
+          region: {enabled: false},
+        },
+      })
+    ).toHaveNoViolations();
```

Comment:
I upgraded `jest-axe` which introduced the `region` rule. I could either roll back that update, or update usages to ignore that rule. Our small examples tend to not have landmark regions, so this rule is not helpful in this context. `jest-axe` doesn't allow global configuration without a central export. We're deprecating the usage of `jest-axe` in favor of `cypress-axe`, so I didn't think updating the 5 usages that are still left would be too bad.